src/types.rs

```diff
@@ -176,7 +181,6 @@ pub struct BundleMeta {
     pub upload_time_epoch: u64,
     pub test_command: Option<String>,
     pub os_info: Option<String>,
-    pub group_is_quarantined: bool,
     pub quarantined_tests: Vec<String>,
```

Comment:
I would put the same info to `quarantined_tests` as we get from the API: classpath, file, parent, name.